### PR TITLE
Final program for PDR

### PR DESCRIPTION
**Pull request by offi-DtrGuo-cial**
Per convention

### CODE REVIEW

File: bs_1/src/telemetry.cpp [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+  uint16_t bl_wheel_speed;
+  uint16_t bl_brake_temperature;
+  uint16_t br_wheel_speed;
+  uint16_t br_brake_temperature;
+
+  uint16_t front_brake_pressure;
+  uint16_t rear_brake_pressure;
+#endif
+
+// #ifdef TELEMETRY_BASE_STATION_RX
```
Comment (izo0619) [inline bs_1/src/telemetry.cpp:92]: Why are these commented out? Remove if not used

File: bs_1/src/telemetry.cpp [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+    can_bus.RegisterRXMessage(fl_wheel_msg);
+    can_bus.RegisterRXMessage(fr_wheel_msg);
+    can_bus.RegisterRXMessage(bl_wheel_msg);
+    can_bus.RegisterRXMessage(br_wheel_msg);
+    can_bus.RegisterRXMessage(brake_pressure_msg);
+
+    can_bus.Initialize(ICAN::BaudRate::kBaud1M);
+  #endif
+
+  #ifdef TELEMETRY_BASE_STATION_RX
```
Comment (izo0619) [inline bs_1/src/telemetry.cpp:151]: Ditto

**Review by izo0619 — CHANGES_REQUESTED** [review]
Looks good for the most part, some small comments. Also, if bs_template is not used/outdated, we should remove it and not merge it. Great documentation BTW

File: bs_1/src/telemetry.cpp (hunk 1 above)
Comment (offi-DtrGuo-cial) [reply bs_1/src/telemetry.cpp:92]: bs_template is used to provide template code for subsequent projects, to automate copy-pasting. bs_1 is not the PDR project; lora_can_2 is. The commit "Final program for PDR" is the one I wanted to make the PR for; does Github not keep track of when the PR is made in between commits?

**Review by izo0619 — APPROVED** [review]

**Pull request merged**